OpenShift/helm/nexus-iq/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "iqserver.fullname" . }}-test-connection"
+  labels:
+    {{- include "iqserver.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args: ['{{ include "iqserver.fullname" . }}:{{ .Values.iq.applicationPort }}']
+  restartPolicy: Never
```

Comment:
ooh, interesting? how's this used? (I have some guesses.)

Reply:
I'm not 100% sure tbh, it comes with the stock templates from 'helm create <projectname>'